packages/lib/src/smartthings-command.ts

```diff
@@ -124,21 +167,24 @@ export abstract class SmartThingsCommand extends Command implements SmartThingsC
 		this._flags = flags
 
 		this._profileName = flags.profile || 'default'
-		this._profileConfig = cliConfig.getProfile(flags.profile)
 
-		let compact = true
-		if ('compactTableOutput' in this.profileConfig) {
-			compact = this.profileConfig.compactTableOutput
-		}
-		if (this.flags.expanded) {
-			compact = false
-		} else if (this.flags.compact) {
-			compact = true
-		}
+		this.cliConfig = await loadConfig({
+			configFilename: `${this.config.configDir}/config.yaml`,
+			managedConfigFilename: `${this.config.configDir}/config-managed.yaml`,
```

Comment:
We should store the managed config in the `cacheDir` instead.

Reply:
Moved to `cacheDir` instead.